src/d3a/models/strategy/utils.py

```diff
@@ -0,0 +1,38 @@
+"""
+Copyright 2018 Grid Singularity
+This file is part of D3A.
+
+This program is free software: you can redistribute it and/or modify it under the terms of the GNU
+General Public License as published by the Free Software Foundation, either version 3 of the
+License, or(at your option) any later version.
+
+This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with this program. If not,
+see <http://www.gnu.org/licenses/>.
+"""
+import numpy as np
+
+from d3a.constants import DEFAULT_PRECISION, RELATIVE_STD_FROM_FORECAST_ENERGY
+
+
+def alter_energy(
```

Comment:
The name gives impression that this function will `slightly` change something (the energy), however it basically receives energy value and `compute` the relative std from it without changing it, so a name starting by compute/calculate or any other option seem better choices imo